src/components/EggManager.js

```diff
@@ -11,17 +11,20 @@ class EggManager extends Component {
     super(props);
     this.state = {
       showModal: false,
-      selectedFriendId: -1,
```

Comment:
It looks like this line was deleted in your pull request, but it also seems like you/we still need this variable in changeDisplayedEggs?